AEPMessaging/Sources/Messaging.swift

```diff
@@ -454,19 +506,13 @@ public class Messaging: NSObject, Extension {
         let requestedSurfaces = surfaces.filter { $0.isValid }
 
         guard !requestedSurfaces.isEmpty else {
-            Log.debug(label: MessagingConstants.LOG_TAG, "Unable to retrieve feed messages, no valid surface paths found.")
+            Log.debug(label: MessagingConstants.LOG_TAG, "Unable to retrieve propositions, no valid surface paths found.")
             dispatch(event: event.createErrorResponseEvent(AEPError.invalidRequest))
             return
         }
 
-        // get propositions from rules engine where conditions are met by this event
-        let ruleConsequencePropositions = getPropositionsFromFeedRulesEngine(event)
-        // get cached propositions
-        var requestedPropositions = retrieveCachedPropositions(for: requestedSurfaces)
```

Comment:
`retrieveCachedPropositions` gets the cached code based propositions. Let's add this back